var/spack/repos/builtin/packages/msmc2/package.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+# @author:>-----Robert QIAO (DRS, Flinders University, Australia)
```

Comment:
We usually don't have a author line here.

Reply:
I see, will this be considered an optional @alalazo? Mainly for us to keep track for the maintenance purpose?

Reply:
If it is for maintenance we have the possibility to add a `mainteiners` class attribute. Basically you can add:
```
maintainers = ['robqiao']
```
under the class definition and we'll use that to ping you whenever there are changes to this recipe. Maintainers are also shown in the output of `spack info <pkg>`.

Reply:
No worries. Will comply. Thx @alalazo